### ISSUE TITLE
Feature Request - Add IPSW

### ISSUE THREAD

**Issue by cdog2020** [scriptingosx/DownloadFullInstaller#2]
It would be awesome if the app also let you fetch and download the IPSW's for Big Sur too.  That and Monterey once available.

**Comment by scriptingosx** [comment]
I agree it would be awesome. However, the known xml feed for IPSWs is an elaborate xml file with entries for every M1 hardware model that all point to the same download url. It doesn't contain multiple previous versions like the software update catalog does. (Even though Apple does still leave the previous versions available for download.)

The way this works right now, I think Mr Macintosh's page is more useful than what I can do in the app:

https://mrmacintosh.com/apple-silicon-m1-full-macos-restore-ipsw-firmware-files-database/

This might change with future hardware and macOS updates though. Worth keeping an eye on.

**Comment by Dowser101** [comment]
Hello, i don’t know if it’s the right place to report some problem...
I tried to fetch the last Big Sur installer and the brand new updated Monterrey.
In both cases the dl went to the end of the 12+GB, and got stuck. 
The only thing i can do is abort the dl, clicking the cross on the right side of the completed dl blue bar.
Nothing appeared in the Applications folder.
Should i look somewhere else  for those massive files ?
My user folder is not on my internal system disk. I have relocated it thru the advanced option of the User preference pane.
Could it be the problem...?  
Thanks to help
